tests/transforms/test_broadcast_expand.py

```diff
@@ -152,14 +173,16 @@ def cost(*params):
 
         assert all(qml.math.allclose(_jac, e_jac) for _jac, e_jac in zip(jac, exp_jac))
 
+    @pytest.mark.slow
     @pytest.mark.tf
-    @pytest.mark.parametrize("params, size", parameters_and_size)
+    @pytest.mark.parametrize("params", parameters)
     @pytest.mark.parametrize("obs, exp_fn", observables_and_exp_fns)
-    def test_tf(self, params, size, obs, exp_fn):
+    def test_tf(self, params, obs, exp_fn):
         """Test that the expansion works with TensorFlow and is differentiable."""
         import tensorflow as tf
 
         params = tuple(tf.Variable(p, dtype=tf.float64) for p in params)
+        dev = qml.device("default.qubit", wires=2)
```

Comment:
Not parametrizing over different `gradient_fn` options here because this is by far the slowest part of this test file.